settings/PandoraSettings_Neutrino_SBND.xml

```diff
@@ -344,6 +344,58 @@
             <tool type = "LArThreeDOpeningAngleFeatureTool"/>
         </FeatureToolsNoChargeInfo>
     </algorithm>
+
+  <!-- Recursively Repeat MopUpAlgorithms -->
+  <algorithm type = "LArRecursivePfoMopUp">
+    <PfoListNames>TrackParticles3D ShowerParticles3D</PfoListNames>
+    <MaxIterations>10</MaxIterations>
+    <MopUpAlgorithms>
+      <algorithm type = "LArBoundedClusterMopUp">
+        <PfoListNames>ShowerParticles3D</PfoListNames>
+        <DaughterListNames>ClustersU ClustersV ClustersW</DaughterListNames>
+      </algorithm>
+      <algorithm type = "LArConeClusterMopUp">
+        <PfoListNames>ShowerParticles3D</PfoListNames>
+        <DaughterListNames>ClustersU ClustersV ClustersW</DaughterListNames>
+      </algorithm>
+      <algorithm type = "LArNearbyClusterMopUp">
+        <PfoListNames>ShowerParticles3D</PfoListNames>
+        <DaughterListNames>ClustersU ClustersV ClustersW</DaughterListNames>
+      </algorithm>
+      <algorithm type = "LArSlidingConePfoMopUp">
+        <InputPfoListNames>TrackParticles3D ShowerParticles3D</InputPfoListNames>
+        <DaughterListNames>ClustersU ClustersV ClustersW TrackClusters3D ShowerClusters3D</DaughterListNames>
+      </algorithm>
+      <algorithm type = "LArSlidingConeClusterMopUp">
+        <InputPfoListNames>TrackParticles3D ShowerParticles3D</InputPfoListNames>
+        <DaughterListNames>ClustersU ClustersV ClustersW</DaughterListNames>
+      </algorithm>
+    </MopUpAlgorithms>
+  </algorithm>
+
+  <algorithm type = "LArBdtPfoCharacterisation">
+    <TrackPfoListName>TrackParticles3D</TrackPfoListName>
+    <ShowerPfoListName>ShowerParticles3D</ShowerPfoListName>
+    <UseThreeDInformation>true</UseThreeDInformation>
+    <MvaFileName>PandoraMVAs/PandoraBdt_v08_33_00_SBND.xml</MvaFileName>
+    <MvaName>PfoCharBDT</MvaName>
+    <MvaFileNameNoChargeInfo>PandoraMVAs/PandoraBdt_v08_33_00_SBND.xml</MvaFileNameNoChargeInfo>
```

Comment:
I note that this is the path for running in sbndcode/larsoft, and not necessarily the path you'd want to specify when running in Pandora standalone, having cloned LArMachineLearningData. We would typically modify here and so put up with a non-zero diff between the XML files for running in standalone and in sbndcode. This diff would be removal of the `PandoraMVAs/` part here (we obviously don't want to change anything about the way this runs in sbndcode/larsoft). Other diffs in our XML files are: inclusion of event reading algorithm for Pandora standalone, and whether the monitoring is on/off by default.

Reply:
Yes, I had done this in the other instances of where the MVAs were called previously but was a bit over zealous in my copy and pasting here. I have pushed a fix to this now.